docs/content/access-api-rate-limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Flow Access API rate limits on Dapper Labs Access nodes
+---
+
+#### Rate limits
+
+Following are the current rate limits for the [Access Node gRPC API](/access-api) in total across all Dapper Labs Access nodes.
```

Comment:
Since the load balancer is operating on requests from all clients, if 2 clients are sending requests at the same time, it's possible their requests are not evenly distributed. Worst case, all requests could go to a single node and they hit the rate limit after 500 requests, and best case they are evenly distributed and hit it after 2000. I think in practice, users will run into some variation is the effective limits, so I tried to clarify that below.

```suggestion
Following are the current rate limits for the [Access Node gRPC API](/access-api) in total across all Dapper Labs Access nodes. The actual limits are enforced by each individual node. If you are using a load balanced endpoint, the limits below are the max rate, and may vary depending on how your requests are routed.
```

Reply:
make sense